build/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM centos:7
 MAINTAINER Ansible Service Broker Community
 
-RUN useradd -M -s /usr/sbin/nologin -u 1001 -g 0 ansibleapp
+RUN mkdir /opt/ansibleapp
+RUN useradd -M -b /opt/ansibleapp -s /usr/sbin/nologin -u 1001 -g 0 ansibleapp
+RUN chown -R 1001:0 /opt/ansibleapp
+
+RUN yum -y update \
+ && yum -y install epel-release centos-release-openshift-origin \
+ && yum -y install origin origin-clients net-tools bind-utils ansible \
```

Comment:
I believe the deploy.sh was using the `ansibleplaybookbundle/ansible-service-broker-asb` so it never really ran the local one IMO.